lib/routes/hlju/news.ts

```diff
@@ -0,0 +1,181 @@
+
+        list.push({
+            title,
+            link: link.startsWith('http') ? link : `${baseUrl}/${link}`,
+            pubDate: dateMatch ? parseDate(dateMatch[0].replaceAll('/', '-')) : undefined,
+        });
+    });
+
+    // 限制返回数量并去重
+    const uniqueList = list.filter((item, index, arr) => arr.findIndex((i) => i.link === item.link) === index).slice(0, 15);
+
+    // 获取每篇文章的详细内容
+    const items = await Promise.all(
+        uniqueList.map((item) =>
+            cache.tryGet(item.link, async () => {
+                // 跳过外部链接
+                if (!item.link.includes('hdxw.hlju.edu.cn')) {
+                    return {
+                        title: item.title,
+                        link: item.link,
+                        description: '外部链接，请点击查看原文',
+                        pubDate: item.pubDate,
+                    };
+                }
+
+                try {
+                    const detailResponse = await ofetch(item.link);
+                    const $detail = load(detailResponse);
+
+                    // 提取文章内容
+                    let description = '';
+
+                    // 尝试多种内容选择器
+                    const contentSelectors = ['.v_news_content', '.news_content', '.article-content', '.content', 'table[width="692"] td', 'td[valign="top"]'];
+
+                    for (const selector of contentSelectors) {
+                        const content = $detail(selector);
+                        if (content.length > 0) {
+                            // 清理内容
+                            content.find('script, style, .nav, .breadcrumb').remove();
+                            const htmlContent = content.html();
+                            if (htmlContent) {
+                                description = htmlContent;
+                                break;
+                            }
+                        }
+                    }
+
+                    // 如果没有找到特定的内容区域，尝试提取主要文本内容
+                    if (!description) {
+                        const bodyContent = $detail('body').text();
+                        const paragraphs = bodyContent
+                            .split('\n')
+                            .map((p) => p.trim())
+                            .filter((p) => p.length > 20 && !p.includes('版权所有') && !p.includes('关闭窗口'))
+                            .slice(0, 5);
+                        description = paragraphs.map((p) => `<p>${p}</p>`).join('');
+                    }
+
+                    // 提取发布时间
+                    let pubDate = item.pubDate;
+                    if (!pubDate) {
+                        const timeText = $detail('body').text();
+                        const dateMatch = timeText.match(/(\d{4})[年/-](\d{1,2})[月/-](\d{1,2})[日\s]/);
+                        if (dateMatch) {
+                            pubDate = parseDate(`${dateMatch[1]}-${dateMatch[2].padStart(2, '0')}-${dateMatch[3].padStart(2, '0')}`);
+                        }
```

Comment:
The date string is already padded on the webpage. It also includes the precise time. Please include them when parsing.